### PR TITLE
Feature/gramine build

### PR DESCRIPTION
**Pull request by james-toussaint**

### CODE REVIEW

File: for-developers/confidential-computing/choose-your-tee-framework.md [hunk 1]
```diff
@@ -1,11 +1,48 @@
 # Choose your TEE framework
 
+{% hint style="warning" %}
+Please make sure you have already checked the [quickstart](../your-first-app.md) and [Your first application](../your-first-app.md) tutorials before building your trusted application on iExec.
```
Comment (jbern0rd) [inline for-developers/confidential-computing/choose-your-tee-framework.md]: Fix link and reuse terms used in the navigation menu on the left ?
```suggestion
Please make sure you have already checked the [Quick Start](../quick-start-for-developers.md) and [Build your first application](../your-first-app.md) tutorials before building your trusted application on iExec.
```

File: for-developers/confidential-computing/create-your-first-gramine-app.md [hunk 2]
```diff
@@ -1,8 +1,198 @@
...
+```
+
+{% endcode %}
+{% endtab %}
+{% endtabs %}
+
+
+Build the docker image.
+
+```sh
```
Comment (jbern0rd) [inline for-developers/confidential-computing/create-your-first-gramine-app.md]: Is there a difference between `bash` and `sh` here ? Should we try to always use the same ?
```suggestion
```bash
```

File: for-developers/confidential-computing/create-your-first-gramine-app.md [hunk 3]
```diff
@@ -1,8 +1,198 @@
...
+docker build . --tag tee-hello-world:1.0.0
+```
+
+Push your image on DockerHub:
+```sh
+docker.io/<username>/tee-hello-world:1.0.0
```
Comment (jbern0rd) [inline for-developers/confidential-computing/create-your-first-gramine-app.md]: Won't there be a problem if `<username>` is not part of the previous `docker build` command ?

Open points (to discuss for consistency across the documentation):
* Use the same conventions in gramine and scone workflows (`<username>` or `<dockerusername>`)
* Help user to distinguish between `gramine`, `scone` and `native` OCI images (avoid `tee-hello-world:1.0.0` generic name ?)

```suggestion
docker build . --tag <dockerusername>/tee-gramine-hello-world:1.0.0
docker push <dockerusername>/tee-gramine-hello-world:1.0.0
```

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: If it is the same, maybe we don't really care

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 4]
```diff
@@ -7,13 +7,8 @@
 - [Nodejs](https://nodejs.org) 14.0.0 or higher.
 - [iExec SDK](https://www.npmjs.com/package/iexec) 8.0.0 or higher.
 - Familiarity with the basic concepts of [Intel® SGX](intel-sgx-technology.md#intel-r-software-guard-extension-intel-r-sgx) and [SCONE](intel-sgx-technology.md#scone-framework) framework.
-  {% endhint %}
-
-{% hint style="warning" %}
-Please make sure you have already checked the [quickstart](../your-first-app.md) and [Your first application](../your-first-app.md) tutorials before building your trusted application on iExec.
```
Comment (jbern0rd) [inline for-developers/confidential-computing/create-your-first-sgx-app.md:13]: I don't know if it wouldn't be a good idea to keep this pre-requisite on first tutorials in build with scone and build with gramine as well.
I know it adds redundancy, but this hint seems really valuable to me.
What do you think ?

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 2 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: We should use the same everywhere yes, I suggest we harmonize them all in a dedicated next PR where we use `bash` everywhere, is it ok?

Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: yes, it is

Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: Local harmonization here https://github.com/iExecBlockchainComputing/documentation/pull/115/commits/f18218a5fa0e20ad5871bd27ff4b66b89ac230f8

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 3 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: Please see https://github.com/iExecBlockchainComputing/documentation/pull/115/commits/56472b2298fafa91fbea4fb74d7db696859ddb4b

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 4 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-sgx-app.md:13]: Yes I agree, please see https://github.com/iExecBlockchainComputing/documentation/pull/115/commits/e809a45d5f656feb2138a4d2227268a27999596a

Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-sgx-app.md:13]: Thanks

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 2 above)
Comment (jbern0rd) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: Thanks

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**